src/DB/SupplierProductRepository.php

```diff
@@ -65,7 +65,19 @@ public function syncProducts(Supplier $supplier, string $mutation, string $count
 			$updates = \array_fill_keys($updates, null);
 
 			foreach (\array_keys($updates) as $name) {
-				$updates[$name] = new Literal("IF((supplierContentLock = 0 && VALUES(supplierLock) >= supplierLock) || fk_supplierContent='$supplierId', VALUES($name), $name)");
+				$updates[$name] = new Literal("IF
+				(
+					(
+						supplierContentLock = 0 && 
+						(
```

Comment:
(VALUES(supplierLock) >= supplierLock && (supplierContentMode = 'length' || (

Líbí se mi to jak si to naformátoval. Je to super pochpopitelný, ale proč v první podmínce máš supplierContentMode = 'length'